_posts/2021-08-24-devs-be-like.md

```diff
@@ -0,0 +1,70 @@
+
+## Getting a job
+
+Getting your first job as a developer can either be easy as heck or the most stressful thing in your entire year. When I was in university, my friends who had time, energy, and interest to do serious hobby projects got job offers at job fairs just by showing what they can do. The rest of us were left feeling (at least I was) overwhelmed and a little scared. 
+
+For a junior developer, having some hobby projects to show can be that crucial thing to land that first job. I never had anything special or grand in my Github. I had some css stuff and, if I recall correctly, a very sad skeleton of a project. I definitely could've invested more in those. But for me, they were there just to show possible employers, that I've done _something_. Since I joined Solita in 2017, excluding some university courses, I haven't done _any_ programming whatsoever that I wasn't paid to do. 
+
+Most employers want some kind of proof of a person's skills as a developer before making any decisions about hiring. Therefore, maintaining some projects or such are most probably an asset when thinking about switching jobs. Personally, I just can't find the motivation to do any of that. Maybe, if I wasn't this content at Solita and felt the need to find another job, I might actually do something about it.
+
+## Keeping your job
+
+No job in the world is static and stays the same. And software development is definitely not a field to change slowly. Being a professional developer you need to keep up with the new stuff. But that's the great part about this job! There is so much to learn and so many ways to develop your own skills further. 
+
+But that does not mean that you need to use your own free time to do all that learning. And obviously, if you want to make a more drastic move in your professional path, you might need to invest some of your own time to learn the basics of that new competence area. But I think that normal project work or time allocated by your company should provide you with enough chances to learn those new, hot and hip programming languages, frameworks, etc.
+
+And besides, software development is definitely not all about programming. Communication is super important. Good people skills come in very handy when you're part of a team. At least someone in the project should have pretty solid skills in coordination and organization. Each team needs a variety of skills and not every developer needs to be the best in programming. They can be a solid 8- (in Finland we are graded from 4 to 10, 10 being the best), but have good people skills. Or they can be a Jira-wizard and use some of their time organizing the team's backlog.
```

Comment:
I would turn around the last sentence about Jira-wizardess. I think the relevant skill is the organization skill and interest to organizing work instead of knowing a tool such as Jira.

... Or they might be really good at organizing work and use some of their time structuring it to epics and tasks int the team's Jira backlog.

Reply:
Good point, will fix 👍